### PR TITLE
Update monitoring.md

### PR DESCRIPTION
**Pull request by YaphetKG**
Adding Frequently Asked questions section for telemetry documentation.

### CODE REVIEW

**Review by cbizon — APPROVED** [review]
Looks good - I know it's not completely general, but in the ingoing/outgoing question, would it make sense to show the lines from aragorn to say "if you're using x it looks like this."  ?

**Comment by YaphetKG** [comment]
> Looks good - I know it's not completely general, but in the ingoing/outgoing question, would it make sense to show the lines from aragorn to say "if you're using x it looks like this." ?

Yep totally makes sense, adding that in.

File: docs/deployment-guide/monitoring.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Given the distributed nature of Biomedical Translator knowledge processing compo
 #### When I instrument my application should I trace incoming requests or outgoing ?
  In a microservices environment such as Translator system, tracing both incoming and outgoing requests is key. Incoming tracing reveals user journeys, while outgoing tracing uncovers dependencies between services—both are crucial for comprehensive visibility and issue diagnosis.
 
+ As an example ARAGORN, an ARA that recieves requests from the ARS and preforms subsequent requests to downstream components makes use of FastAPI instrumentation to trace incoming requests and httpx instrumentation for tracing outgoing requests. This [code snippet](https://github.com/ranking-agent/aragorn/blob/main/src/otel_config.py) shows how ARAGORN traces both incoming and outgoing requests.
```
Comment (cbizon) [inline docs/deployment-guide/monitoring.md:20]: preforms -> performs

Comment (codewarrior2000) [reply docs/deployment-guide/monitoring.md]: Also needs this correction, recieves -> receives  (rule: "i" before "e", except after "c")

Comment (YaphetKG) [reply docs/deployment-guide/monitoring.md]: Thank you both for the pointers, found a couple of other places where things were not correct.

**Comment by edeutsch** [comment]
It is easier to read the draft here:
https://github.com/NCATSTranslator/TranslatorTechnicalDocumentation/blob/telemetry-FAQ/docs/deployment-guide/monitoring.md

Note that the code block controls don't seem to be working as intended

**Comment by YaphetKG** [comment]
> It is easier to read the draft here: https://github.com/NCATSTranslator/TranslatorTechnicalDocumentation/blob/telemetry-FAQ/docs/deployment-guide/monitoring.md
> 
> Note that the code block controls don't seem to be working as intended

HTML in there was casting the markdown. i think its looking good now. Thanks for pointing that out @edeutsch.

**Pull request merged**